test/angular/angular-wizard.test.ts

```diff
@@ -131,5 +131,21 @@ describe('angular-wizard', () => {
         }
       `);
     });
+
+    it('uses dummy DSN "http://test:0000" in spotlight mode', () => {
```

Comment:
h: just to confirm: the `http://test:0000` DSN is not actually injected into user files, correct?

If so, I'd strongly suggest _not_ to add this DSN but instead set an empty string or `undefined`. If no (valid) DSN is found, the SDK will fall back to checking `spotlight` anyway, so this just more clearly communicates the intention that the SDK is set up purely for spotlight.